.travis.yml

```diff
@@ -6,6 +6,19 @@ branches:
 
 language: objective-c
 
+env:
+  matrix:
+    - SDK=iphonesimulator8.1 OS=8.1 NAME=iphone 6
+    - SDK=iphonesimulator8.2 OS=8.2 NAME=iphone 6
+    - SDK=iphonesimulator8.3 OS=8.3 NAME=ihpone 6
+    - SDK=iphonesimulator8.4 OS=8.4 NAME=iphone 6
```

Comment:
IMHO we only need 7.x (until we drop it), 8.y, and 9.z (with x,y,z the most recent patch number)